boards/DCM/Src/App/states/App_DriveState.c

```diff
@@ -27,13 +28,18 @@ static void DriveStateRunOnTick100Hz(struct StateMachine *const state_machine)
     App_AllStatesRunOnTick100Hz(state_machine);
 
     struct DcmWorld *world = App_SharedStateMachine_GetWorld(state_machine);
-    struct DcmCanRxInterface *can_rx = App_DcmWorld_GetCanRx(world);
+    struct DcmCanRxInterface *can_rx      = App_DcmWorld_GetCanRx(world);
+    struct ErrorTable *const  error_table = App_DcmWorld_GetErrorTable(world);
```

Comment:
```suggestion
    struct ErrorTable *error_table = App_DcmWorld_GetErrorTable(world);
```
Just for consistency with `error_table` code